_meta/beat.yml

```diff
@@ -4,13 +4,13 @@
 
 apm-server:
   # Defines the host and port the server is listening on
-  host: "localhost:8080"
 
-  #server.max_unzipped_size:
-  #server.max_header_bytes:
-  #server.read_timeout: 2s
-  #server.write_timeout: 2s
+  #host: "localhost:8080"
```

Comment:
I am +1 on removing the `server` namespace, as it seems redundant within `apm-server`.